docs/manual/src/udl/ext_types_wrapped.md

```diff
@@ -57,13 +83,83 @@ Because `UniffiCustomTypeWrapper` is defined in each crate, this means you can w
 if they are not defined in your crate - see the 'wrapper_types' example which demonstrates
 how `serde_json::Value` can be wrapped.
 
-## Custom Foreign Wrappers
+## Bindings example
+
+Suppose you have Rust struct with recently renamed fields:
+
+```rust
+pub struct User {
+  full_name: String,
+  age: Int32,
+}
+```
+
+But your Kotlin code is still using the old field names:
+
+```kotlin
+data class LegacyUser(
+  var name: String,
+  var years_old: Int,
+}
+```
+
+We again start by declaring this type in our UDL via a `typedef` with a `Wrapped` attribute.
+
+```idl
+[Wrapped]
+typedef User LegacyUser;
+```
+
+This time however, we will update the Kotlin section of the `uniffi.toml` file:
+
+```toml
+[bindings.kotlin.wrappers.LegacyUser]
+# Name of the Kotlin class that will wrap `User`
+type_name = "LegacyUser"
+# Function that converts `User` to `LegacyUser`
+wrap = "wrapUser"
+# Function that converts `LegacyUser` to `User`
+unwrap = "unwrapUser"
```

Comment:
Swift is really tricky.  I implemented it using unnamed params, but I'm not sure if that's good or bad.  Placeholders would also cut down on some boilerplate, like being able to use `{}.toString()` on Kotlin rather than having to create a separate function to do that.

I removed them just because I wanted to avoid anything code-like in the `.toml` file, but I'll try adding them back.

Reply:
Cool. I like that you have some flexibility (e.g. you've got `{}.toLogin()` `{}.toLogin()` for the Logins example you gave earlier).